moxin-backend/src/lib.rs

```diff
@@ -10,13 +10,16 @@ pub struct Backend {
 
 impl Default for Backend {
     fn default() -> Self {
-        Backend::new()
+        Backend::new(".".to_string())
```

Comment:
The expectation is that the frontend can assign a path to cache downloaded models, such as `$HOME/.moxin/models`.